en/scripting/subpackage.md

```diff
@@ -1,49 +1,49 @@
-# 代码分包加载
+# Code Sub-packaging
 
-随着游戏玩法越来越丰富，游戏的代码量也越来越大，开发者对于扩大包大小的需求越来越强烈，同时微信小游戏也支持了分包加载的功能，所以 Cocos Creator 推出了代码分包加载这样一个功能。
+As gameplay becomes more and more rich, the amount of code in the game is increasing and developers are wanting to expand the size of their game. **Code sub-packaging** provides for these demands.
```

Comment:
"expand the size of their game **package**" maybe batter than "expand the size of their game"?